.github/workflows/tck-test.yml

```diff
@@ -37,10 +37,15 @@ jobs:
       working-directory: test_agent/java
       run: |
             mvn clean install --file pom.xml
-    - name: Set up Python 3.8.7
+    - name: Install Conan
```

Comment:
is there any way we could test this out by triggering one CPP test through the workflow?